integration/common.sh

```diff
@@ -29,6 +29,20 @@ waitForServerStartup() {
   sleep 2
 }
 
+pickUnusedPort() {
+  local base=$1
```

Comment:
Suggestion: Have a hard-coded base and make pickUnusedPort() parameterless?

If we have a free port we don't care whether it's >= some arbitrary number. Any port is good.